test/test_cromwell_workflow.sh

```diff
@@ -9,38 +9,41 @@ INPUTS_JSON=$4
 WDL_FILE=$5
 OPTIONS_FILE=$6
 DEPENDENCIES_JSON=$7
+# Read list of dependency files into an array to pass to cromwell-tools
+DEPENDENCIES_LIST=$(cat ${DEPENDENCIES_JSON} | python3 -c "import json,sys;obj=json.load(sys.stdin);print(' '.join(obj.values()));")
 
 echo "Starting workflow."
-WORKFLOW_HASH=$(cromwell-tools run \
+WORKFLOW_HASH=$(cromwell-tools submit \
   --username "${CROMWELL_USER}" \
   --password "${CROMWELL_PASSWORD}" \
-  --cromwell-url "${CROMWELL_URL}" \
-  --inputs-json "${INPUTS_JSON}" \
+  --url "${CROMWELL_URL}" \
+  --inputs_files "${INPUTS_JSON}" \
   --wdl-file "${WDL_FILE}" \
   --options-file "${OPTIONS_FILE}" \
-  --dependencies-json "${DEPENDENCIES_JSON}" \
+  --deps-file ${DEPENDENCIES_LIST[@]} \
```

Comment:
What will DEPENDENCIES_LIST look like?
You might want to double-quote it like the other arguments?
--deps-file "${DEPENDENCIES_LIST[@]}" \

Reply:
If there are quotes around `DEPENDENCIES_LIST` I believe it will get passed in as one string, but what we want is `--deps-file "file1.txt" "file2.txt" "file3.txt"` which is why I left out the quotes.